src/styles/index.scss

```diff
@@ -1,3 +1,87 @@
-body {
+/* Глобальные настройки */
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the correct font is imported from Google Fonts as specified in the task description. This was mentioned in the previous review.